crates/bevy_render/src/camera/visible_entities.rs

```diff
@@ -204,8 +204,8 @@ pub fn visible_entities_system(
         &mut VisibleEntities,
         Option<&RenderLayers>,
     )>,
-    visible_query: Query<(Entity, &Visible, Option<&RenderLayers>)>,
-    visible_transform_query: Query<&GlobalTransform, With<Visible>>,
+    visible_query: Query<(Entity, &Visible, Option<&RenderLayers>), Without<OutsideFrustum>>,
+    visible_transform_query: Query<&GlobalTransform, Without<OutsideFrustum>>,
```

Comment:
Double negatives 🤯

_(nothing we can do here, but always causes me to pause for a moment when reading logic like this)_

Reply:
Yeah I like to avoid double negatives in apis whenever possible. But yeah the tradeoff here for the inverse is too much :)